sqlmesh/core/loader.py

```diff
@@ -413,10 +429,10 @@ def _load_sql_models(
         jinja_macros: JinjaMacroRegistry,
         audits: UniqueKeyDict[str, ModelAudit],
         signals: UniqueKeyDict[str, signal],
+        cache: CacheBase,
```

Comment:
Should we consider making this a non-breaking change by making this optional?

Reply:
discussed offline, chances are low folks override this, additionally, it will break unless they pre-emptively added kwargs to their override